packages/roosterjs-editor-api/lib/utils/processList.ts

```diff
@@ -64,7 +66,10 @@ function applyListFormat(node: Node, formats: string[], isDarkMode: boolean) {
         {
             fontFamily: formats[0],
             fontSize: formats[1],
-            textColor: formats[2],
+            textColors: {
+                lightModeColor: formats[4],

```

Comment:
formats[4] can be null. Do you think we should skip textColors when formats[4] is null?

Reply:
Actually, you highlighted a problem with this whole block here, this is definitely not the right way to go. I'm gonna refactor this real quick.